dashboard/tests/test_dashboard.py

```diff
@@ -2,21 +2,34 @@
 import json
 import time
 import logging
+import asyncio
 
+import aiohttp.web
 import ray
 import psutil
 import pytest
 import redis
 import requests
 
 from ray import ray_constants
-from ray.test_utils import wait_for_condition, wait_until_server_available
+from ray.test_utils import (
+    wait_for_condition,
+    wait_until_server_available,
+    run_string_as_driver,
+)
 import ray.new_dashboard.consts as dashboard_consts
+import ray.new_dashboard.utils as dashboard_utils
 import ray.new_dashboard.modules
 
+try:
+    create_task = asyncio.create_task
+except AttributeError:
+    create_task = asyncio.ensure_future
```

Comment:
Why do we need this?

Reply:
This is for `test_async_loop_forever`.

Reply:
Hmm.. I am still confused. Can you elaborate a little more and add comments too? Is there any way we can remove this at all?

Reply:
Sorry, `test_async_loop_forever` use `loop.create_task` not `asyncio.create_task`. This 4 lines can be removed, I will fix it.